src/main/_docs/integration-guide/integration-continuous-integration.md

```diff
@@ -7,41 +7,56 @@ permalink: /:categories/continuous-integration/
 ---
 {% include base.html %}
 
-Codenvy is connected to your repo so any change made to the repo that would normally trigger a CI job will continue to trigger a CI job when the change is made in Codenvy.
+Codenvy can use [Factories]({{base}}{{site.links["factory-getting-started"]}}) with a Jenkins CI system to generate developer workspaces pre-configured with the context of the CI job. For example, a failed CI build email can be customized to include a link to two factories that open a workspace pre-configured to:
+* The branch and commit ID that broke the build for the project. 
+* The head of the master branch for the project.
```

Comment:
HEAD of branch. master is a branch too. So, if it is master then it is head of master.